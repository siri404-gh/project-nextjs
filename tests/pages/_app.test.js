import { render, unmountComponentAtNode } from 'react-dom';
import { act } from 'react-dom/test-utils';

import App from '@/pages/_app';

describe('App', () => {
  let container;

  beforeEach(() => {
    container = document.createElement('section');
    document.body.appendChild(container);
  });

  afterEach(() => {
    unmountComponentAtNode(container);
    container.remove();
    container = null;
  });

  it('Should render without props', () => {
    act(() => {
      render(<App />, container);
    });

    expect(container).toMatchSnapshot();
  });

});
