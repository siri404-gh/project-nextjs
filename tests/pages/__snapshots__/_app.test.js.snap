// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Should render without props 1`] = `
<section>
  <header
    class="MuiPaper-root MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary makeStyles-appBar-4 mui-fixed MuiPaper-elevation0"
  >
    <div
      class="MuiToolbar-root MuiToolbar-regular makeStyles-toolbar-10 MuiToolbar-gutters"
    >
      <button
        aria-hidden="true"
        class="MuiButtonBase-root MuiIconButton-root Navbar__leftbutton makeStyles-leftBtn-5"
        tabindex="0"
        type="button"
      >
        <span
          class="MuiIconButton-label"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-colorSecondary"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
            />
          </svg>
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
      <div
        class="Navbar__titlebar makeStyles-titlebar-7"
      >
        <div
          class="makeStyles-titleLeft-6"
        />
        <button
          aria-controls="simple-menu"
          aria-hidden="true"
          class="MuiButtonBase-root MuiIconButton-root"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root makeStyles-small-11 MuiSvgIcon-colorSecondary"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 3c1.66 0 3 1.34 3 3s-1.34 3-3 3-3-1.34-3-3 1.34-3 3-3zm0 14.2c-2.5 0-4.71-1.28-6-3.22.03-1.99 4-3.08 6-3.08 1.99 0 5.97 1.09 6 3.08-1.29 1.94-3.5 3.22-6 3.22z"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
  </header>
  <div
    class="PrivateSwipeArea-root-17 PrivateSwipeArea-anchorLeft-18"
    style="width: 20px;"
  />
  <div
    class="makeStyles-contentWrapper-2 layout"
  >
    <div
      class="makeStyles-shift-3"
    />
    <div
      class="makeStyles-shift-3"
    />
  </div>
</section>
`;
